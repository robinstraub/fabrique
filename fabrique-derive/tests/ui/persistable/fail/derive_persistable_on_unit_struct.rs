use fabrique_derive::Persistable;

#[derive(Persistable)]
struct Anvil;

fn main() {}
