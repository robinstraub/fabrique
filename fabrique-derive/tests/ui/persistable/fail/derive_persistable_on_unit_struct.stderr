error: Factory can only be derived from named structs, unit struct given
 --> tests/ui/persistable/fail/derive_persistable_on_unit_struct.rs:4:1
  |
4 | struct Anvil;
  | ^^^^^^
